C++/source/searching/Exponential_search.cpp

```diff
@@ -0,0 +1,66 @@
+// C++ program to find an element x in a
+// sorted array using Exponential search.
+#include <bits/stdc++.h>
```

Comment:
This does not work for all compilers. Include only those headers that are required.